tests/test_optical_model.py

```diff
@@ -0,0 +1,32 @@
+from catkit2.catkit_bindings import DataStream
+from catkit2.simulator import SimpleOpticalModel
+
+import hcipy
+import numpy as np
+import time
+
+
+def test_optical_model():
+    model = SimpleOpticalModel()
+    pupil_grid = model.pupil_grid
+
+    wf_in = hcipy.Wavefront(pupil_grid.ones())
+    model.set_wavefronts('pre_pupil', wf_in)
+    
+    test_wf = model.get_wavefronts('science_camera')[0]
+    size1 = test_wf.intensity.shaped.shape[0]
+    size2 = test_wf.intensity.shaped.shape[1]
+
+    stream = DataStream.create('images', 'mod', 'float64', [size1, size2], 20)
```

Comment:
I'm not sure why the data stream is needed here. This is a leftover from very early on, when I wanted to test an early version of the camera viewer UI.